### PR TITLE
Enhancements and fixes for the Authorization of Web Endpoints section

### PR DESCRIPTION
**Pull request by MichalMaler**
PR with little updates and fixes for the reviewed section.
Closing: https://issues.redhat.com/browse/QDOCS-114

Signed-off-by: Michal Maléř <mmaler@redhat.com>

### CODE REVIEW

**Review by jmartisk — APPROVED** [review]

**Review by sberyozkin — APPROVED** [review]

**Comment by MichalMaler** [comment]
Rebased, squashed. Well done, We!

File: docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc [hunk 1]
```diff
@@ -163,9 +163,9 @@ quarkus.http.auth.permission.roles2.policy=admin-policy1
...
 TIP: Given the above permission set, `GET /api/foo` would match both permission sets' paths, requiring both the `user` and `admin` roles.
 
-=== Configuration Properties to Deny access
+=== Configuration properties to deny access
 
-The following configuration settings alter the RBAC Deny behavior:
+The following configuration settings alter the role-based access control (RBAC) denying behavior:
 
 `quarkus.security.jaxrs.deny-unannotated-endpoints=true|false`::
 If set to true, the access will be denied for all JAX-RS endpoints by default, so if a JAX-RS endpoint does not have any security annotations, it will default to the `@DenyAll` behavior.
```
Comment (michelle-purcell) [inline docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc:171]: Suggestion to reduce sentence length:

```suggestion
If set to true, access is denied for all JAX-RS endpoints by default. If a JAX-RS endpoint does not have any security annotations, it defaults to the `@DenyAll` behavior.
```

**Review by michelle-purcell — APPROVED** [review]
@MichalMaler - Looks great. Thanks :+1:  Just one minor suggestion.

File: docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 [id="security-authorization-of-web-endpoints-reference"]
```
Comment (michelle-purcell) [inline docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc:6]: @MichalMaler - Was just about to link to this great page from the security overview PR and realized I missed this suggestion:

Would it make sense to reduce length of filename to:

 `security-authorize-web-endpoints-reference.adoc` 

and the respective ID:

```suggestion
[id="security-authorize-web-endpoints-reference.adoc"]
```

Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc:6]: @michelle-purcell Yeah, why not :) Sure. Do you want me to do it in this PR or we can merge and you will do it in yours? What is better in your books?

Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc]: Renamed the file, changed the ID and fixed the xrefs :)

Comment (MichalMaler) [reply docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc]: Renaming patch here:
https://github.com/quarkusio/quarkusio.github.io/pull/1616

**Comment by sberyozkin** [comment]
Nice work Michal, thanks

File: docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc (hunk 2 above)
Comment (michelle-purcell) [reply docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc]: @MichalMaler - It's your baby! ... Thanks. Looks great :+1:

**Pull request merged**